app/_data/speakers_info.yml

```diff
@@ -0,0 +1,129 @@
+- firstName: Avdi
+  lastName: Grimm
+  shortDesc: Founder of RubyTapas
+  twitter: avdi
+  github: avdi
+  bio: Avdi Grimm has been hacking Ruby code for over 10 years, and is still loving it. He is chief aeronaut at ShipRise, head chef at RubyTapas.com, and a Ruby Rogue. He lives in Eastern Tennessee with his wife and a steadily incrementing number of children, and in his copious spare time blogs and podcasts at Virtuous Code and Wide Teams.
+  talk_title: TBA
+  talk_description: TBA
+  picture_name: avdi.png
+  website:
```

Comment:
I guess you needed all the info in this new yml, and not just the bio and talk description. That makes sense, and I wasn't thinking correctly at first

But rather than having this duplicated, It would be better to just have a single speakers file, so I'd move all new info to the original speakers.yml

Or maybe that's just a matter of replacing speakers.yml with this one?